packages/mui-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/base",
-  "version": "5.0.0-beta.69",
+  "version": "5.0.0-beta.70",
```

Comment:
Releasing base is safe: Last release was also alongside v6: https://github.com/mui/material-ui/pull/45077/files#diff-24e31acbbc79c41518034f9e4ef908afca76c2f502ad03134baa5e0534eacc76

We need to release to fix the `@mui/types` dependency